JPS_Ontology/ontology/ontobuiltenv/OntoBuildingEnvironment.csv

```diff
@@ -118,3 +118,10 @@ representativeFor,Object Property,,,AveragePricePerSqm,PostalCode,,A relationshi
 representativeFor,Object Property,,,AveragePricePerSqm,PostalCode,exactly 1,,https://www.theworldavatar.com/kg/ontobuiltenv/,representative for
 Bank,Class,Non-Domestic,IS-A,,,,A property where customers can deposit and withdraw money.,https://www.theworldavatar.com/kg/ontobuiltenv/,Bank
 Pharmacy,Class,Medical,IS-A,,,,A healthcare facility where medicinal drugs are prepared or sold.,https://www.theworldavatar.com/kg/ontobuiltenv/,Pharmacy
+BuildingFloors,Class,,,Property,,,A concept to describe amount of floors in a building.,https://www.theworldavatar.com/kg/ontobuiltenv/,Building floors
+hasBuildingFloors,Object Property,,,Property,BuildingFloors,exactly 1,A relationship between a property and its amount floors.,https://www.theworldavatar.com/kg/ontobuiltenv/,has building floors
+hasFloorsValue,Data Property,,,BuildingFloors,integer,exactly 1,A relationship between a building floors and its value.,https://www.theworldavatar.com/kg/ontobuiltenv/,has floors value
```

Comment:
"a building floors " does not seem grammatically correct to me

Reply:
it's class name 'BuildingFloors'